Haruby-iOS/Sources/Manager/HarubyCalculateManager.swift

```diff
@@ -26,7 +26,7 @@ final class HarubyCalculateManager {
         
         // 2. 현재 날짜로부터 endDate까지 DailyBudget을 순차적으로 방문
         for dailyBudget in salaryBudget.dailyBudgets {
-//            if dailyBudget.date > now { continue }
+            if dailyBudget.date > now { continue }
```

Comment:
부등호가 다음과 같이 변경되어야 할 거 같습니다
```suggestion
            if dailyBudget.date < now { continue }
```